src/celeritas/field/detail/FieldUtils.hh

```diff
@@ -62,10 +62,9 @@ inline CELER_FUNCTION Chord make_chord(Real3 const& src, Real3 const& dst)
         result.dir[i] = dst[i] - src[i];
     }
     result.length = norm(result.dir);
-    const real_type norm = 1 / result.length;
     for (int i = 0; i < 3; ++i)
     {
-        result.dir[i] *= norm;
+        result.dir[i] /= result.length;
```

Comment:
1 div + 3 mult + 1 variable vs. 3 div : is the latter more performant?  Actually I like the more terse/minimal code here.

Reply:
I saw this while cleaning up: it's extra code that's possibly a "premature optimization" that might even be a pessimization on GPU since it might require an extra 64-bit register...